### PR TITLE
Extra tests for deployment-flow ssh

### PR DESCRIPTION
**Pull request by Lukewh**

### CODE REVIEW

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-gce.jujugui.org:8080//job/juju-gui/8214/

File: jujugui/static/gui/src/app/components/deployment-flow/sshkey/test-sshkey.js [hunk 1]
```diff
@@ -297,6 +297,36 @@ describe('DeploymentSSHKey', function() {
       );
     });
 
+    it ('removes key from table', () => {
```
Comment (hatched) [inline jujugui/static/gui/src/app/components/deployment-flow/sshkey/test-sshkey.js]: space

**Comment by Lukewh** [comment]
Yeah no worries @hatched

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-gce.jujugui.org:8080//job/juju-gui/8216/

**Comment by Lukewh** [comment]
:shipit:

**Pull request merged**